### ISSUE TITLE
Action to Send to PAFL bus

### ISSUE THREAD

**Issue by michaelroper** [bitfocus/companion-module-allenheath-avantis#7]
Hi @Bambii556, I saw you mentioned in another issue that you’re doing some rewrites to this module at the moment - are you able to see if it’s possible to add an action to send an input to either of the PAFL busses on the Avantis?

Thanks!

**Comment by Bambii556** [comment]
Hi @michaelroper, I will take a look if possible with the current Midi command, as I don't see it anywhere in their docs.

**Comment by michaelroper** [comment]
Yeah, I had a look at the doc as well  after I posted that, agree it didn’t look promising.. :/ 

For my use case I might be able to assign it to a soft key on the desk instead, if you do happen to have any luck with implementing issue #6
